core/preprocessing/preprocess_dmi.py

```diff
@@ -0,0 +1,51 @@
+from core.util.io import read_csv, write_csv
+import pandas as pd
+
+
+def remove_timeformatting() -> None:
+    """Clean DMI data and save to disk."""
+    original = read_csv("external/w_d_codes.csv")
+
+    # Split 'observed' into 'Date' and 'Time'
+    original["Date"] = original["observed"].str.split("T").str[0]
+    time = original["observed"].str.split("T").str[1]
+
+    # Reformat 'Date' from YYYY-MM-DD to DD-MM-YYYY
+    original["Date"] = original["Date"]
+    # Clean 'Time': remove trailing "00Z" and adjust leading zeros
+    time = time.str.replace("Z", "", regex=False)
+    # Set 'Time' in original
+    original["Time"] = time
+
+    # Drop the original 'observed' column
+    original = original.drop(columns=["observed"])
+
+    # Save cleaned data
+    write_csv(original, "interim/interim_dmi.csv")
+
+
+def dmi_split_weathertype() -> None:
+    """Split the weather type column into multiple columns and normalize the data."""
```

Comment:
This is splitting a single file into multiple - one for each weather_type, and not "into multiple columns", right?